spyder/plugins/explorer/widgets.py

```diff
@@ -851,7 +875,177 @@ def create_func(fname):
 
     def go_to_parent_directory(self):
         pass
-        
+
+    def copy_path(self, fnames=None, method="absolute"):
+        """Copy absolute or relative path to given file(s)."""
+        cb = QApplication.clipboard()
+        home_directory = getcwd_or_home()
+        if fnames is None:
+            fnames = self.get_selected_filenames()
+        if not isinstance(fnames, (tuple, list)):
+            fnames = [fnames]
+        fnames = [_fn.replace(os.sep, "/") for _fn in fnames]
+        if len(fnames) > 1:
+            if method == "absolute":
+                clipboard_files = ''.join('"' + _fn + '",' + '\n' for _fn in
+                                          fnames)
+            elif method == "relative":
+                clipboard_files = ''.join('"' +
+                                          osp.relpath(_fn, home_directory).
+                                          replace(os.sep, "/") + '",' +
+                                          '\n' for _fn in fnames)
+            clipboard_files = "[" + clipboard_files[:-2] + "]"

```

Comment:
I suggest you not insert the list brackets here and let the user choose the collection type instead by simply typing it themselves (like the quote character with the single path) since they may want to insert it into a different type of collection/structure, as discussed in the main review comments.